.github/scripts/strategy-matrix/generate.py

```diff
@@ -150,23 +158,58 @@ def generate_strategy_matrix(all: bool, config: Config) -> list:
             config_name += f'-{n}'
         if (n := os['compiler_version']) != '':
             config_name += f'-{n}'
-        config_name += f'-{architecture['platform'][architecture['platform'].find('/')+1:]}'
+        config_name += f'-{architecture["platform"][architecture["platform"].find("/")+1:]}'
         config_name += f'-{build_type.lower()}'
         if '-Dunity=ON' in cmake_args:
             config_name += '-unity'
 
         # Add the configuration to the list, with the most unique fields first,
         # so that they are easier to identify in the GitHub Actions UI, as long
-        # names get truncated.
-        configurations.append({
-            'config_name': config_name,
-            'cmake_args': cmake_args,
-            'cmake_target': cmake_target,
-            'build_only': build_only,
-            'build_type': build_type,
-            'os': os,
-            'architecture': architecture,
-        })
+        # names get truncated. Add Address and Thread (both coupled with UB) sanitizers when the distro is bookworm.
+        if os['distro_version'] == 'bookworm':
+            if cxx_flags:
+                cmake_args_flags = f'{cmake_args} \"-DCMAKE_CXX_FLAGS=-fsanitize=address,{sanitizers_flags} -fno-omit-frame-pointer {cxx_flags} -O1\"'
```

Comment:
```suggestion
                cmake_args_flags = f'{cmake_args} -DCMAKE_CXX_FLAGS="-fsanitize=address,{sanitizers_flags} -fno-omit-frame-pointer {cxx_flags} -O1"'
```

I think this is the correct way to write this

Reply:
This leads to another issue in the final resolution of this string in the `.github/workflows/reusable-build.yml` file. To show you what it is, it will be better if I just make the change and let the script run and generate the error.

Reply:
Didn't solve the issue. I think some CI script is doing some string manipulation on these commands. Leading to this.
https://github.com/XRPLF/rippled/actions/runs/19233877555/job/54978695284?pr=5996